### PR TITLE
[Logs] Update Instant Logs filters

### PR DESCRIPTION
**Pull request by cicku**

### CODE REVIEW

File: content/logs/instant-logs.md [hunk 1]
```diff
@@ -12,29 +12,27 @@ Instant Logs allows Cloudflare customers to access a live stream of the traffic
 
 {{<feature-table id="analytics.instant_logs">}}
 
-## Instant Logs on the Cloudflare dashboard
+## Instant Logs via Cloudflare Dashboard
 
-1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login).
+1. Select the [zone](/fundamentals/setup/accounts-and-zones/#zones) you want to use with Instant Logs.
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select the [zone](/fundamentals/setup/accounts-and-zones/#zones) you want to use with Instant Logs.
```

File: content/logs/instant-logs.md [hunk 2]
```diff
@@ -12,29 +12,27 @@ Instant Logs allows Cloudflare customers to access a live stream of the traffic
...
 
-2. Select the domain you want to use with Instant Logs.
+2. Go to **Analytics** > **Instant Logs**.
 
-3. Go to **Analytics** > **Instant Logs**.
+3. Click **Start streaming**.
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
3. Select **Start streaming**.
```

File: content/logs/instant-logs.md [hunk 3]
```diff
@@ -12,29 +12,27 @@ Instant Logs allows Cloudflare customers to access a live stream of the traffic
...
 
-4. Select **Start streaming**.
+4. (optional) Click **Add filter** to narrow down the events to be shown.
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
4. (optional) Select **Add filter** to narrow down the events to be shown.
```

File: content/logs/instant-logs.md [hunk 4]
```diff
@@ -12,29 +12,27 @@ Instant Logs allows Cloudflare customers to access a live stream of the traffic
...
 
-5. Select **Add filters** to narrow down the events shown.
+Fields supported in our [HTTP requests dataset](/logs/reference/log-fields/zone/http_requests/) can be used when you add filters. Some fields with additional subscriptions required are not supported in the dashboard, you will need to use CLI instead.
 
-The filters you can add are **ASN**, **Cache status**, **Country**, **Client IP**, **Host**, **HTTP method**, **Path**, **Status code**, **Firewall action matches**, and **Firewall rule ID matches**. If you would like to filter on additional criteria, leave us feedback on the form linked on the Instant Logs page.
+*Once a filter is selected and the stream has started, only log lines that match the filter criteria will appear. Filters are not applied retroactively to logs already showing in the dashboard.*
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
Once a filter is selected and the stream has started, only log lines that match the filter criteria will appear. Filters are not applied retroactively to logs already showing in the dashboard.
```

File: content/logs/instant-logs.md [hunk 5]
```diff
@@ -44,34 +42,24 @@ Instant Logs has a maximum data rate supported. For high volume domains, we samp
...
-| Not Equals               | `"neq"`        |
-| Greater Than             | `"gt"`         |
-| Greater Than or Equal to | `"geq"`        |
-| Less Than                | `"lt"`         |
-| Less Than or Equal to    | `"leq"`        |
-| Starts with              | `"startsWith"` |
-| Ends with                | `"endsWith"`   |
-| Contains                 | `"contains"`   |
-| Is in                    | `"In"`         |
+All supported operators can be found at our [Filters](/logs/reference/filters/) page.
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
All supported operators can be found in the [Filters](/logs/reference/filters/) page.
```

File: content/logs/instant-logs.md [hunk 6]
```diff
@@ -44,34 +42,24 @@ Instant Logs has a maximum data rate supported. For high volume domains, we samp
...
 "filter":"{\"where\":{\"and\":[{\"key\":\"ClientCountry\",\"operator\":\"neq\",\"value\":\"ca\"}]}}"
 ```
 
 ```bash
+# Filter when the status code returned from Cloudflare is either 200 or 201:
 "filter":"{\"where\":{\"and\":[{\"key\":\"EdgeResponseStatus\",\"operator\":\"in\",\"value\":\"200,201\"}]}}"
 ```
 
 ```bash
+# Filter when the request path contains "/static" and the request hostname is "theburritobot.com":
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
# Filter when the request path contains "/static" and the request hostname is "example.com":
```

File: content/logs/instant-logs.md [hunk 7]
```diff
@@ -44,34 +42,24 @@ Instant Logs has a maximum data rate supported. For high volume domains, we samp
...
 "filter":"{\"where\":{\"and\":[{\"key\":\"ClientRequestPath\",\"operator\":\"contains\",\"value\":\"/static\"}, {\"where\":{\"and\":[{\"key\":\"ClientRequestHost\",\"operator\":\"eq\",\"value\":\"theburritobot.com\"}]}}"
```
Comment (angelampcosta) [inline content/logs/instant-logs.md]:
```suggestion
"filter":"{\"where\":{\"and\":[{\"key\":\"ClientRequestPath\",\"operator\":\"contains\",\"value\":\"/static\"}, {\"where\":{\"and\":[{\"key\":\"ClientRequestHost\",\"operator\":\"eq\",\"value\":\"example.com\"}]}}"
```

**Comment by angelampcosta** [comment]
@soheiokamoto or @cmackenzie1  could you also review this PR? Thanks!

File: content/logs/instant-logs.md (hunk 1 above)
Comment (cicku) [reply content/logs/instant-logs.md]: There is pretty much no reason to mention login, everyone has to login to use the feature.

File: content/logs/instant-logs.md (hunk 2 above)
Comment (cicku) [reply content/logs/instant-logs.md]: No big difference, I can change if this really matters.

Comment (angelampcosta) [reply content/logs/instant-logs.md]: It's just to follow our guidelines https://developers.cloudflare.com/style-guide/formatting/keyboard-keys/#keyboard-keys.

File: content/logs/instant-logs.md (hunk 1 above)
Comment (angelampcosta) [reply content/logs/instant-logs.md]: It's a standard we use in the documentation.

**Review by soheiokamoto — APPROVED** [review]

**Pull request merged**